jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/ChipsTests.java

```diff
@@ -32,67 +39,126 @@ public class ChipsTests extends TestsInit {
+        autocompleteChips.is().hasElement(APPLE);
     }
 
-    @Test
+    @Test(description = "Test checks the input field in chips group")
     public void chipsWithInputTest() {
-        chipsWithInputField.show();
-        chipsWithInputField.is().displayed();
-        chipsWithInputInput.is().assertChipsIsEnabled();
-        chipsWithInputInput.has().assertChipsHasPlaceholder(PLACEHOLDER);
-        chipsWithInputInput.input("Kiwi" + "\n");
-        chipsWithInputInput.input("Melon");
-        chipsWithInputInput.clearInputField();
-        chipsWithInputInput.input("Rockmelon" + "\n");
+        inputChips.show();
+        inputChips.is().displayed();
+        inputChips.has().placeholder(PLACEHOLDER);
+        inputChips.input("Kiwi" + "\n");
+        inputChips.input("Melon");
+        inputChips.clearInputField();
+        inputChips.input("Rockmelon" + "\n");
+        inputChips.is().hasElement("Kiwi");
+        inputChips.is().hasElement("Rockmelon");
+    }
+
+    @Test(description = "Test checks that chips can be removed")
+    public void chipsRemovableTest() {
+        inputChips.show();
+        inputChips.is().displayed();
+        inputChips.getElement("Lemon").is().removable();
+        inputChips.getElement("Lemon").close();
+    }
+
+    @Test(description = "Test checks that chips can be focused")
+    public void chipFocusedTest() {
+        inputChips.show();
+        inputChips.is().displayed();
+        inputChips.getElement("Apple").click();
+        inputChips.getElement("Apple").is().focused();
+        inputChips.getElement("Lime").click();
+        inputChips.getElement("Lime").is().focused();
+    }
+
+    @Test(description = "Test checks that chips has avatar image")
+    public void chipAvatarTest() {
+        String expectedSrc = "https://material.angular.io/assets/img/examples/shiba1.jpg";
+        avatarChips.show();
+        avatarChips.getElement("Dog one").leadingIcon().is().src(expectedSrc);
+    }
+
+    @Test(description = "Test checks that chips can be disabled")
+    public void chipDisabledTest() {
+        disabledChips.show();
+        disabledChips.is().displayed();
+        disabledChips.is().disabled();
+    }
+
+    @Test(description = "Test checks chips multiply selection")
+    public void chipsMultipleSelectionTest() {
+        multipleChips.show();
+        multipleChips.is().displayed();
+        multipleChips.is().multiselectable();
+        multipleChips.selectByText("multiple color none");
+        multipleChips.selectByText("multiple color Primary");
+        multipleChips.getElement("multiple color none").is().selected();
+        multipleChips.getElement("multiple color Primary").is().selected();
+    }
+
+    @Test(description = "Test checks that chips has error state")
+    public void chipsErrorState() {
+        errorChips.show();
+        errorChips.is().displayed();
+        errorChips.input("Kiwi" + "\n");
+        errorChips.has().errorState();
     }
 }
```

Comment:
should be empty line at the end

Reply:
@MayaElf  add this rule to the style checker, after that it will be required

Reply:
@pnatashap we have it in CheckStyle 
![image](https://github.com/jdi-testing/jdi-light/assets/55189259/2e56164e-0023-4c75-981f-b2498110fa50)
Sorry, can't find why it does not work.